sdks/python/apache_beam/metrics/cells.py

```diff
@@ -292,7 +292,7 @@ def singleton(value):
     return DistributionData(value, 1, value, value)
 
   def to_runner_api(self):
-    return beam_fn_api_pb2.Metrics.User.DistributionData(
+    return beam_fn_api_pb2.Metrics.PTransform.User.DistributionData(
```

Comment:
I'm just the peanut gallery, of course, but this doesn't seem clearer at all than `beam_fn_api_pb2.DistributionData(...)`